support/locust/README.md

```diff
@@ -30,7 +30,7 @@ The program reads the environment variable `LOCUST_TEST_SRC` for the location of
 LOCUST_HOST='JASECI_URL' LOCUST_TEST_SRC='sample_code/<YOUR TEST>' locust -f run_jac.py
 ```
 
-Go to the link specified in console, e.x http://0.0.0.0:8089 and specify the desired number of users for the load test and initiate the test. You can also change the server URL in the web UI.
+Go to the link specified in console, e.x http://0.0.0.0:8089 and specify the desired number of users for the load test and initiate the test. Please make sure that the host url is the same with LOCUST\_HOST.
```

Comment:
Is this true? Can I not change the URL in the UI?

Reply:
No you cannot change the URL in UI. If the two URLs are different, there might be unexpected behavior. This is because the preparation stage is prior to the launch of web UI. I need to know the URL beforehand so that I can register the sentinel and upload the source code and load the actions. This might be a bit confusing tho...